### PR TITLE
508: [COGNITIION, SCREEN READER] Add aria-attributes to sign in button

### PR DESCRIPTION
**Pull request by Mottie**
## Description

The form `cta-widget` (call to action widget) and `SaveInProgressIntro` components render a button to either prompt the Veteran to sign in or start the form. All content below the button may be ignored by a screen reader user. To fix this, the a11y experts recommended adding an `aria-describedby` to the button (or action link) to inform screen reader users that there is more content. See option 1 in https://github.com/department-of-veterans-affairs/va.gov-team/issues/16182.

The aforementioned components may also render the `ProgressButton` component from the component-library. This is updated in https://github.com/department-of-veterans-affairs/component-library/pull/193.

Note: `aria-label` was also included as part of a separate set of a11y issues - see https://github.com/department-of-veterans-affairs/va.gov-team/issues/18923

## Original issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/16182

## Testing done

Added unit tests

## Screenshots

<details><summary>a11y recommendation</summary>

Intro page screenshot with added text showing the sign in button with aria-describedby and matching ID on the header below the call to action</details>

## Acceptance criteria
- [x] `aria-label` and `aria-describedby` added to call to action widget buttons
- [x] `aria-label` and `aria-describedby` added to save in progress intro buttons
- [x] All tests passing
- [x] Passes axe tests

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/static-pages/cta-widget/components/messages/SignIn.jsx [hunk 1]
```diff
@@ -2,20 +2,28 @@ import React from 'react';
...
     heading: `Please sign in to ${serviceDescription}`,
     headerLevel,
     alertText: (
       <p>
-        Try signing in with your <b>DS Logon</b>, <b>My HealtheVet</b>, or{' '}
-        <b>ID.me</b> account. If you don’t have any of those accounts, you can
-        create one now.
+        Try signing in with your <strong>DS Logon</strong>,
+        <strong>My HealtheVet</strong>, or <strong>ID.me</strong> account. If
+        you don’t have any of those accounts, you can create one now.
```
Comment (Mottie) [inline src/applications/static-pages/cta-widget/components/messages/SignIn.jsx:19]: I couldn't just leave the `<b>` tags there :eye-twitch:

File: src/platform/forms/save-in-progress/FormStartControls.jsx [hunk 2]
```diff
@@ -108,6 +111,8 @@ class FormStartControls extends React.Component {
                   ? 'usa-button-primary'
                   : 'usa-button-secondary'
               }
+              ariaLabel={ariaLabel}
+              ariaDescribedby={ariaDescribedby}
             />
           )}
           <Modal
```
Comment (Mottie) [inline src/platform/forms/save-in-progress/FormStartControls.jsx:118]: The `<ProgressButton>`s inside this Modal were not updated on purpose.

File: src/platform/forms/save-in-progress/FormStartControls.jsx [hunk 3]
```diff
@@ -179,6 +188,8 @@ FormStartControls.propTypes = {
       continueAppButtonText: PropTypes.string,
     }),
   }),
+  ariaLabel: PropTypes.string,
+  ariaDescribedby: PropTypes.string,
 };
 
 FormStartControls.defaultProps = {
```
Comment (bkjohnson) [inline src/platform/forms/save-in-progress/FormStartControls.jsx:195]: Could we assign the `null` default props down here?

**Review by bkjohnson — APPROVED** [review]
Looks good, minor comments.

**Review by ndsprinkle — APPROVED** [review]

**Pull request merged**